client/src/components/Add-Edit-Form/index.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Form, Input, Button, message } from 'antd';
+import './style.css';
+
+const axios = require('axios');
+
+const AddEditForm = ({ formType, addLink, editLink, cohortId, inputData }) => {
+  const onFinish = async (values) => {
+    try {
+      const sendValues = values;
+      if (formType !== 'cohort') {
+        sendValues.cohortId = cohortId;
+      }
+
+      if (addLink) {
+        const response = await axios.post(addLink, sendValues);
+        const { message: resMessage } = response.data.data;
+        message.success(resMessage);
+      } else {
+        const response = await axios.put(editLink, sendValues);
+        const { message: resMessage } = response.data.data;
+        message.success(resMessage);
+      }
+    } catch (err) {
+      const { message: errMessage } = err.response.data.data;
```

Comment:
Here maybe throw error ``` Cannot destructure property of undefined``` so before this line you should be check types of errors and handle it depending on the status code.